packages/core/is-v2-ready-yet/.eslintrc

```diff
@@ -1,11 +1,9 @@
 {
   "extends": "@parcel/eslint-config",
-  "plugins": ["react"],
   "env": {
     "browser": true
   },
   "rules": {
-    "react/jsx-uses-vars": "error",
-    "react/jsx-uses-react": "error"
+    "react/prop-types": "off" // TODO: Enable this when this package is flow-typed
```

Comment:
eslint parses configs with json5, so this comment is okay. That said, some other tools don't like it (VSCode complains right away when opening the file). Alternative: could move this to a js or yaml config.